src/patternfly/components/Banner/banner.scss

```diff
@@ -57,4 +57,18 @@
     z-index: var(--pf-c-banner--m-sticky--ZIndex);
     box-shadow: var(--pf-c-banner--m-sticky--BoxShadow);
   }
+
+  .pf-c-button.pf-m-link.pf-m-inline {
```

Comment:
Looks like the styles added are limited to the inline link button component - is that intended @mcarrano or should a user be able to add a regular link, too?

Reply:
I was assuming a regular link.

Reply:
I added that. @mcarrano which examples do you want to show? Do we need all these? I put them there to make sure everything works but I'm not sure if we need to show them all as examples or not.

Reply:
@srambach Yes, I do feel like we have too many examples here. What's the difference between a link and an inline link? I think that the Call to action example should be removed as it's not something we recommend that banners be used for. I'm on the fence about whether we need to allow for both inline text links and link buttons. The banner was originally designed to hold short text messages like an alert. It's reasonable that you might include a text link in that message. A button implies to me some action that probably does not belong on a banner. Put I'm OK to leave it there as there might be a use case I'm not considering.

@mceledonia @lboehling I also want to get your input/feedback about the styling. Looks like the current approach is to make the link text bold and then underline on hover. I'm not sure that bold text affords a link to me. Would just underlining the link be more clear (in it's default state)?

Reply:
Also, although I was able to make the not-allowed cursor on the disabled button link, there are good a11y reasons not to try to make a link look disabled. Also I had to turn pointer events back on to do the cursor change for the link button, which isn't really good to do. So maybe we forget about that part?
And if we don't want to support button links, I can either take that out or leave it in just in case, and remove the examples. 
I'll wait to hear about about the bold/underline.